workflows/amr/run.wdl

```diff
@@ -218,14 +219,16 @@ task RunRedup {
                 f.write("\n")
         CODE
 
-        counter=1
-        for fasta in ~{sep=' ' host_filtered_reads}; do
-            seqtk subseq $fasta duplicated-pairs.txt > redups_$counter.fastq
-            ((counter++))
+        export HOST_FILTERED_READS_FILES=(~{sep=' ' host_filtered_reads})
+        export SUBSAMPLED_READS_FILES=(~{sep=' ' subsampled_reads})
+
+        for index in ${!HOST_FILTERED_READS_FILES[@]}; do
```

Comment:
Could you add some comments so that it's clear what's happening?

Reply:
Added!